.editorconfig

```diff
@@ -1,162 +1,96 @@
-###############################
-# Core EditorConfig Options   #
-###############################
+# editorconfig.org
+# Copied from the .NET Core repo https://github.com/dotnet/corefx/blob/master/.editorconfig
 
+# top-most EditorConfig file
 root = true
 
-# All files
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
 [*]
-indent_style = space
-end_of_line = lf
```

Comment:
I think we should look at end_of_line=lf to make development on linux/mac more straightforward and having a consistent line ending.

Reply:
Isn't Git going to take care of that?

Reply:
Anyway, it's still there, please see https://github.com/dotnet/corefx/blob/master/.editorconfig#L158